api/reference/1.0/notes/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Overview
+---
+
+# Notes
+
+`Notes` are textual annotations you can add to Launch resources of these types:
+- `DataElements`
+- `RuleComponents`
+- `Rules`
+- `Extensions`
+- `Libraries`
+- `Properties`
+
+Those six types are collectively known as the `Notable` resources.
+
+When a `Notable` resource is deleted, its associated `Notes` are also deleted.
+
+## `Notes` are Comments
+Launch doesn't attach any meaning to `Notes`.
+Adding a `Note` won't change the behavior of your `Property`.
+
+You might use `Notes` to:
+- provide background information
+- function as to-do lists
+- pass along resource usage advice
+- give instructions to other team members
+- record historical context
+- remind yourself why this resource is built the way it is
+
+But Launch ignores `Notes` so you are free to use them any way you want.
```

Comment:
@thebenrob Would you please share your thoughts on this section.

Reply:
I think it's fine.  The last sentence I might change for stylistic reasons and say something like "The contents of `Notes` have no impact on the behavior of property resources, so you are free to use them any way you want.

Reply:
I've replaced my last sentence with @thebenrob's. And then used that to replace the paragraph _before_ the list of suggestions, since it said virtually the same thing and there was need to repeat it.